torch/csrc/jit/script/tree_views.h

```diff
@@ -24,6 +24,7 @@ namespace script {
 // Def   = Def(Ident name, List<Param> params, List<Stmt> body)         TK_DEF
 //
 // Stmt  = If(Expr cond, List<Stmt> true_body, List<Stmt> false_body)   TK_IF
+//       | For(List<Ident> targets, List<Expr> itrs, List<Stmt> body)   TK_FOR
```

Comment:
Oh come on, there's no reason to save this one character from `iters` 😄

Reply:
Also, when reading the code above, it's not always clear for me that `targets` are the iterator variables and not e.g. the outputs of the loop. How about `iter_vars`?

Reply:
This is modeled after the Python AST `For` node which names them `targets`: https://docs.python.org/3/library/ast.html

Still change?

Reply:
Idk, you even call them `iter_vars` (or similar) in the code you wrote, and I think that's simpler. But I'm ok with leaving it too